NachoSpigot-Server/src/main/java/net/minecraft/server/EntityMinecartAbstract.java

```diff
@@ -30,9 +30,9 @@
     private double derailedX = 0.5;
     private double derailedY = 0.5;
     private double derailedZ = 0.5;
-    private double flyingX = 0.95;
-    private double flyingY = 0.95;
-    private double flyingZ = 0.95;
+    private double flyingX = 0.949999988079071D; // Paper - restore vanilla precision
+    private double flyingY = 0.949999988079071D; // Paper - restore vanilla precision
+    private double flyingZ = 0.949999988079071D; // Paper - restore vanilla precision
```

Comment:
Reference? and is this the case for 1.8?

Reply:
Yes https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/nms-patches/EntityMinecartAbstract.patch?at=refs%2Fheads%2Fversion%2F1.8#157